tests/integration/tests/test_sanity_checks.py

```diff
@@ -181,7 +181,6 @@ def test_cspm_aws_findings(
         query_list = build_query_list(
             benchmark_id="cis_aws",
             match_type=match_type,
-            version=AGENT_VERSION,
             agent=agent,
```

Comment:
Sanity checks are used for both `agent-based` and `agentless` deployments. Since we don't have control over the agentless agent version deployment, we cannot use this parameter in the query. However, we can still correctly identify the retrieved documents because we use `agent.id` for identification.